spec/authorizers/organization_authorizer_spec.rb

```diff
@@ -1,23 +1,24 @@
 require 'spec_helper'
 
 describe OrganizationAuthorizer do
-  before do
-    @org = Organization.create!(name: 'test')
-  end
+  let(:org) {
+    Organization.create!(name: 'test')
```

Comment:
No time.  The main goal was to attack `Application::create_sample_user` and its kind.